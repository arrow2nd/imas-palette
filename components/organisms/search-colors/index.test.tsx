import { render } from '@testing-library/react'
import SearchColors from './index'

describe('SearchColors', () => {
  test('見た目が変化していないか', () => {
    const { container } = render(
      <SearchColors current="" onChange={jest.fn()} />
    )

    expect(container).toMatchSnapshot()
  })
})
