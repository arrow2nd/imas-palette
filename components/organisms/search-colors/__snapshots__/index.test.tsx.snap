// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchColors 見た目が変化していないか 1`] = `
<div>
  <div
    class="flex flex-wrap mt-8 justify-center items-center"
  >
    <button
      class="mx-1 p-0 rounded-full border-2 border-gray-600"
      data-testid="search-color-button-0"
    >
      <div
        class="w-8 h-8 rounded-full border-2 bg-gradient-to-r from-red-500 to-purple-500"
        title="全て"
      />
    </button>
    <button
      class="mx-1 p-0 rounded-full border-2 border-transparent"
      data-testid="search-color-button-1"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        style="background-color: rgb(255, 0, 0);"
        title="赤"
      />
    </button>
    <button
      class="mx-1 p-0 rounded-full border-2 border-transparent"
      data-testid="search-color-button-2"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        style="background-color: rgb(255, 165, 0);"
        title="橙"
      />
    </button>
    <button
      class="mx-1 p-0 rounded-full border-2 border-transparent"
      data-testid="search-color-button-3"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        style="background-color: rgb(255, 255, 0);"
        title="黄"
      />
    </button>
    <button
      class="mx-1 p-0 rounded-full border-2 border-transparent"
      data-testid="search-color-button-4"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        style="background-color: rgb(144, 226, 0);"
        title="黄緑"
      />
    </button>
    <button
      class="mx-1 p-0 rounded-full border-2 border-transparent"
      data-testid="search-color-button-5"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        style="background-color: rgb(0, 128, 0);"
        title="緑"
      />
    </button>
    <button
      class="mx-1 p-0 rounded-full border-2 border-transparent"
      data-testid="search-color-button-6"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        style="background-color: rgb(0, 255, 255);"
        title="水色"
      />
    </button>
    <button
      class="mx-1 p-0 rounded-full border-2 border-transparent"
      data-testid="search-color-button-7"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        style="background-color: rgb(0, 0, 255);"
        title="青"
      />
    </button>
    <button
      class="mx-1 p-0 rounded-full border-2 border-transparent"
      data-testid="search-color-button-8"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        style="background-color: rgb(128, 0, 128);"
        title="紫"
      />
    </button>
    <button
      class="mx-1 p-0 rounded-full border-2 border-transparent"
      data-testid="search-color-button-9"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        style="background-color: rgb(255, 255, 255);"
        title="白"
      />
    </button>
    <button
      class="mx-1 p-0 rounded-full border-2 border-transparent"
      data-testid="search-color-button-10"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        style="background-color: rgb(0, 0, 0);"
        title="黒"
      />
    </button>
  </div>
</div>
`;
